www/templates/default/html/manager/Events/event-list.tpl.php

```diff
@@ -1,69 +0,0 @@
-<label for="bulk_action">Bulk Actions</label>
```

Comment:
This file is actually necessary because it is being used to render the search results too.

https://github.com/unl/UNL_UCBCN_System/blob/master/www/templates/default/html/manager/Search.tpl.php#L6